vita3k/gui/src/settings_dialog.cpp

```diff
@@ -741,6 +741,10 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Checkbox("Texture Cache", &emuenv.cfg.texture_cache);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Uncheck the box to disable texture cache.");
+        ImGui::SameLine();
+        ImGui::Checkbox("Show Compile Shaders", &emuenv.cfg.show_compile_shaders);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable display of compile shaders dialog.");
```

Comment:
```suggestion
            ImGui::SetTooltip("Uncheck the box to disable the display of compile shaders dialogs.");
```
I'm not entirely sure about this.

Reply:
is wrong, is only one dialog ^^

Reply:
but i have pushed fix typo for the missing

Reply:
It's one dialog which shows up multiple time so both are right I guess. If it's only one dialog it should be "display of the compile shader dialog."  I think.

Reply:
is because it show count of shaders :) so shaers but dialog
one dialog for few shaders :)

Reply:
Ah sorry, that was a typo, I did not meant to tell you to replace shaders by shader.

Reply:
no problem :)